crates/iota-grpc-api/README.md

```diff
@@ -0,0 +1,46 @@
+# IOTA gRPC API
+
+This crate introduces a gRPC API for IOTA. The primary goal of this API is to provide a more efficient and lower-latency method for data access, intended to replace existing REST-API polling or filesystem-based synchronization. This reduces the delay between data creation and their subsequent processing by external services.
```

Comment:
One last thing, we should mention that this is highly experimental, more like an internal thing, and is about to chance with high chance, so people shouldn't use it yet.

Reply:
Agreed. Thanks for pointing this out, I added a small paragraph in the beginning of the doc at [f01ae56](https://github.com/iotaledger/iota/pull/7511/commits/f01ae56410e24a1924803e99fd5cebc6aa92058c), so in the future we might just remove it when it is production ready.